proposals/target-typed-new.md

```diff
@@ -76,13 +68,12 @@ Action a = new(); // no constructor found
 
 ### Miscellaneous
 
-`throw new()` is allowed.
-
-Target-typed `new` is not allowed with binary operators.
-
-It is disallowed when there is no type to target: unary operators, collection of a `foreach`, in a `using`, in a deconstruction, in an `await` expression, as an anonymous type property (`new { Prop = new() }`), in a `lock` statement, in a `sizeof`, in a `fixed` statement, in a member access (`new().field`), in a dynamically dispatched operation (`someDynamic.Method(new())`), in a LINQ query, as the operand of the `is` operator, as the left operand of the `??` operator,  ...
+The following are consequences of the specification:
 
-It is also disallowed as a `ref`.
+- `throw new()` is allowed (the target type is `System.Exception`)
+- Target-typed `new` is not allowed with binary operators.
```

Comment:
The restriction on binary operators is not really a consequence of above spec. It is a separate spec rule, no?

Reply:
Yes, that is in iteration 3.